src/main/java/vazkii/botania/common/item/relic/ItemDice.java

```diff
@@ -69,14 +75,31 @@ public ItemDice(Properties props) {
 				}
 			}
 
-			if (possible.isEmpty()) {
-				player.sendMessage(new TranslationTextComponent("botaniamisc.dudDiceRoll", world.rand.nextInt(6) + 1).mergeStyle(TextFormatting.DARK_GREEN), Util.DUMMY_UUID);
-				stack.shrink(1);
-				return ActionResult.resultSuccess(stack);
-			} else {
+			if (!possible.isEmpty()) {
 				int relic = possible.get(world.rand.nextInt(possible.size()));
 				player.sendMessage(new TranslationTextComponent("botaniamisc.diceRoll", relic + 1).mergeStyle(TextFormatting.DARK_GREEN), Util.DUMMY_UUID);
 				return ActionResult.resultSuccess(new ItemStack(getRelics()[relic]));
+			} else {
+				int roll = world.rand.nextInt(6) + 1;
```

Comment:
hardcoding the number of tables here is meh. Can we just have a dice/bonus_roll table that dispatches to the 6 subtables using weighted entries?

Reply:
I wanted to actually tie the roll to the table used, a minor and inconsequential detail, I guess I'll cut that though.

Reply:
o lmao